static/js/zcommand.js

```diff
@@ -84,6 +84,10 @@ exports.process = function (message_content) {
         return true;
     }
 
+    if (content === '/settings') {
+        window.location.hash = 'settings/your-account';
```

Comment:
Hm, not sure if this is the correct thing here. One part of me says this works (though seems hacky). Is there a way to open the modal with a function call?

Also, remember to `return true;`.

Reply:
Yeah, at first, this seems hacky to me too(I guessed `overlays.open_settings` does that but it only open overlays), but I think this should work properly because we have our own event handler for `hashchange` which handles this properly and I also think this is the only way to make this work.
@showell can clear this!

Reply:
I agree with @shubhamdhama. I tried doing this with a function call, but It was working. (Mainly by calling `settings.setup_page()`). Even in [settings menu](https://github.com/zulip/zulip/blob/master/static/js/settings_panel_menu.js#L70) this is being done.